### PR TITLE
Insert received data at CloudRun Endpoint into BQ 

### PR DESCRIPTION
**Pull request by PradeepSingh1988**
## Pull Request Template
Please, go through these steps before you submit a PR.

1. Make sure that your PR is not a duplicate.
2. **Before** submitting the PR, review your changes:
    * Adjust the code to the existing [style](https://google.github.io/styleguide).
    * Add a set of unit tests for the changes and ensure they all pass.
    * Add accompanying README.md file with instructions on usage. See [awesome-readme](https://github.com/matiassingers/awesome-readme) for good examples of high-quality READMEs.
    * Add a link to your contribution in the top-level [README](https://github.com/GoogleCloudPlatform/professional-services/blob/master/README.md) (alpha-order).
    * Add [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) license headers with an up-to-date copyright date attributed to `Google LLC`.
    * Remove unnecessary LICENSE files. There's no need to include an additional license since all repository submissions are covered by the top-level Apache 2.0 [license](https://github.com/GoogleCloudPlatform/professional-services/blob/master/LICENSE).
    * **For new tools/examples**, file your project with the PSO Engineering Council.

3. **After** these steps, you're ready to open a pull request.
    * Give a descriptive title to your PR.
    * Provide a description of your changes.
    * Include the Engineering Council bug ID with your pull request.
    * Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
<br/>

**PLEASE REMOVE THIS TEMPLATE BEFORE SUBMITTING**

### CODE REVIEW

**Comment by boredabdel** [comment]
Few comments. The source files are missing the header license https://opensource.google/documentation/reference/releasing/licenses

**Comment by PradeepSingh1988** [comment]
> Few comments. The source files are missing the header license https://opensource.google/documentation/reference/releasing/licenses

@boredabdel , added license text to all files.

**Comment by boredabdel** [comment]
@PradeepSingh1988 there are still few unresolved comments

**Review by boredabdel — COMMENTED** [review]
Added few comments. Sorry i forgot to hit submit

File: examples/clouddrun-to-bq/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Use the official lightweight Python image.
```
[the remark this answers is not in the document]
Comment (PradeepSingh1988) [reply examples/clouddrun-to-bq/Dockerfile]: Done

File: examples/clouddrun-to-bq/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# https://hub.docker.com/_/python
+FROM python:3.10-slim
```
[the remark this answers is not in the document]
Comment (PradeepSingh1988) [reply examples/clouddrun-to-bq/Dockerfile]: Done

File: examples/clouddrun-to-bq/cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+steps:
```
[the remark this answers is not in the document]
Comment (PradeepSingh1988) [reply examples/clouddrun-to-bq/cloudbuild.yaml]: Done

File: examples/clouddrun-to-bq/main.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
+"""
```
[the remark this answers is not in the document]
Comment (PradeepSingh1988) [reply examples/clouddrun-to-bq/main.py]: Done

File: examples/clouddrun-to-bq/main_test.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+"""
```
[the remark this answers is not in the document]
Comment (PradeepSingh1988) [reply examples/clouddrun-to-bq/main_test.py]: Done

File: examples/clouddrun-to-bq/requirements-test.txt [hunk 6]
```diff
@@ -0,0 +1,2 @@
+pytest==7.1.1
+pylint
```
[the remark this answers is not in the document]
Comment (PradeepSingh1988) [reply examples/clouddrun-to-bq/requirements-test.txt]: Done

File: examples/clouddrun-to-bq/requirements.txt [hunk 7]
```diff
@@ -0,0 +1,6 @@
+Flask==2.1.0
```
[the remark this answers is not in the document]
Comment (PradeepSingh1988) [reply examples/clouddrun-to-bq/requirements.txt]: Done

File: examples/clouddrun-to-bq/requirements.txt [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+gunicorn==20.1.0
+google-cloud-bigquery==1.8.1
+google-cloud-core==0.29.1
+google-cloud-logging
+jsonschema
```
[the remark this answers is not in the document]
Comment (PradeepSingh1988) [reply examples/clouddrun-to-bq/requirements.txt]: Done

**Review by boredabdel — APPROVED** [review]
LGTM

**Pull request merged**